src/main/frontend/format/mldoc.cljs

```diff
@@ -154,7 +154,7 @@
           properties (assoc properties :tags tags :alias alias)
           properties (-> properties
                          (update :filetags (constantly filetags)))
-          properties (medley/filter-kv (fn [_k v] (seq v)) properties)]
+          properties (medley/remove-kv (fn [_k v] (or (nil? v) (and (seq? v) (empty? v)))) properties)]
```

Comment:
better to use `coll?` here instead of `seq?`, for example, the value of `(seq? [:a :b])` is false.

Reply:
`()` introduced by `tags` and `filetags`.
`#{}` introduced by multi-value properties.

`coll?` catches the above situation all.